File: 231208/특정 숫자 도달하기/reaching-specific-number.py
arr = list(map(int, input().split()))

sum = 0
avg = 0
cnt = 0 
for i in arr :
    if i <= 250 :
        sum += i
        cnt += 1
    if i >= 250 :
        break

avg = sum / cnt
print(sum, "{:.1f}".format(avg))

File: 240112/특정 숫자 도달하기/reaching-specific-number.js
// 배열을 구현하여 주어진 수를 입력받습니다.
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split(" ");
input = input.map(Number) ;

let sum_val = 0 ;
let cnt = 0 ;

for (let e of input) {
    if (e >= 250) {
        break ;
    }
    sum_val += e ;
    cnt++ ;
}

let result = sum_val / cnt ;
console.log(sum_val, result.toFixed(1)) ;